src/Http/Response.php

```diff
@@ -2041,6 +2042,66 @@ public function withCookie($name, $data = '')
     }
 
     /**
+     * Create a new response with an expired cookie set.
+     *
+     * ### Options
+     *
+     * - `name`: The Cookie name
+     * - `value`: Value of the cookie
+     * - `expire`: Time the cookie expires in
+     * - `path`: Path the cookie applies to
+     * - `domain`: Domain the cookie is for.
+     * - `secure`: Is the cookie https?
+     * - `httpOnly`: Is the cookie available in the client?
+     *
+     * ### Examples
+     *
+     * ```
+     * // set scalar value with defaults
+     * $response = $response->withoutCookie('remember_me');
+     *
+     * // customize cookie attributes
+     * $response = $response->withoutCookie('remember_me', ['path' => '/login']);
+     *
+     * // add a cookie object
+     * $response = $response->withoutCookie(new Cookie('remember_me', 'deleted'));
+     * ```
+     *
+     * @param string|\Cake\Http\Cookie\Cookie $name The name of the cookie to expire, or a cookie object
+     * @param array $options An array of cookie options.
+     * @return static
+     */
+    public function withoutCookie($name, $options = [])
```

Comment:
Is this method 'removing' a cookie from the response that is in flight or adding an expired cookie? Right now `withoutCookie()` will add a cookie which is a bit counter-intuitive.